<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VotingWidget</class>
 <widget class="QWidget" name="VotingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="ownerLabel">
     <property name="text">
      <string>Owner</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="optionBtn1">
     <property name="text">
      <string>option1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="optionBtn2">
     <property name="text">
      <string>option2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="optionBtn3">
     <property name="text">
      <string>option3</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="optionBtn4">
     <property name="text">
      <string>option4</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="optionBtn5">
     <property name="text">
      <string>option5</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
